### PR TITLE
Fileidsfix

### PR DESCRIPTION
**Pull request by emimaesmith**
Reverting the file_ids access statement and adding logging statements

### CODE REVIEW

File: scale/job/data/job_data.py [hunk 1]
```diff
@@ -295,26 +295,33 @@ def extend_interface_with_inputs_v5(self, interface, job_files):
         inputs = []
         input_files = deepcopy(interface.get_input_files())
         input_json = deepcopy(interface.get_input_json())
+        method = 'scale.job.data.job_data.JobData.extend_interface_with_inputs_v5'
+        logger.info("%s::Converting seed interface for job %s to v5", method, interface.get_name())
```
Comment (gisjedi) [inline scale/job/data/job_data.py]: We probably want both `name` and `jobVersion` here. As well as just debug level.

File: scale/job/data/job_data.py [hunk 2]
```diff
@@ -295,26 +295,33 @@ def extend_interface_with_inputs_v5(self, interface, job_files):
...
+        logger.info("%s::Number of incoming job_files: %i", method, len(job_files))
```
Comment (gisjedi) [inline scale/job/data/job_data.py]: Debug as well.

File: scale/job/data/job_data.py [hunk 3]
```diff
@@ -295,26 +295,33 @@ def extend_interface_with_inputs_v5(self, interface, job_files):
...
             # Follow that up with a list comprehension over potentially multiple IDs to get
             # final list of ScaleFile objects
             if in_file['name'] not in self._new_data.values:
+                logger.info("%s::Cannot find %s in file values, skipping input file", method, in_file['name'])
                 continue
-            in_file['value'] = [file_map[x] for x in self._new_data.values[in_file['name']['file_ids']]]
+            in_file['value'] = [file_map[x] for x in self._new_data.values[in_file['name']].file_ids]
 
             if len(in_file['value']) >= 2:
+                logger.info("%s::in_file[\'value\'] has at least two file_ids", method)
```
Comment (gisjedi) [inline scale/job/data/job_data.py]: Debug

File: scale/job/data/job_data.py [hunk 4]
```diff
@@ -295,26 +295,33 @@ def extend_interface_with_inputs_v5(self, interface, job_files):
...
                 in_file['type'] = 'files'
             else:
+                logger.info("%s::in_file['value'] has %i file_ids", method, len(in_file['value']))
```
Comment (gisjedi) [inline scale/job/data/job_data.py]: Debug

**Comment by emimaesmith** [comment]
Removed logging since we're testing onsite

**Pull request merged**